docs/UserGuide.md

```diff
@@ -112,45 +102,46 @@ Will eventually be extended to get help about specific commands.
 - `addr` - adds a new recipe
 
 **SYNOPSIS:**
-- `addr` **-n**  \<**product name**\> **-items** **\<item name\[quantity\]**, … > \[-pc \<num>\]
-\[-d \<desc\>\]
+- `addr` **-n \<product name\>** **-items \<item name\[quantity\], … >** \[-pc \<num>\] \[-d \<desc\>\]
 
 **DESCRIPTION:**
 - **product name:**	name of the item created by the recipe
 - **-items:** specify the list of material items used
 - **-pc:** quantity of product produced in a craft (default: 1)
 - **-d:** description of recipe (default: “No description given”)
+- Adds a recipe to the inventory, with the given fields
 
 **EXAMPLE:**
 - `addr` <u>Bob’s anvil</u> **-items** <u>block of iron</u> \[3\], <u>iron ingot</u>\[4\]
 - Adds a **recipe** to craft <u>Bob’s anvil</u>, which takes 3 <u>blocks of iron</u> and 4 <u>iron ingots</u>
 
 
 ### Adding a item: `add`
-NOT Supported as of v1.2 yet
+NOT Supported as of v1.2
 
 **NAME:**
 
-- `add` - add a quantity to a single specified item
+- `add` - adds quantity to a single specified item
```

Comment:
is now supported as addq, should update this?

Reply:
Done, my bad